packages/app-desktop/utils/preprocessor.ts

```diff
@@ -0,0 +1,17 @@
+const preprocessor = {
+    // Open the details tag
+    openDetailsTags: (htmlData: string): string => {
```

Comment:
What if I have the details tag inside a markdown code block? Like this
\```
\<details>
...
\</details>
\```

Reply:
It won't add if it is under backticks

<img width="808" alt="Screenshot 2022-03-13 at 4 08 40 AM" src="https://user-images.githubusercontent.com/71817691/158037438-90be2d2c-1043-46d9-a875-8cdfbfd3df4a.png">
<img width="813" alt="Screenshot 2022-03-13 at 4 08 51 AM" src="https://user-images.githubusercontent.com/71817691/158037442-c0790664-3d8f-4b41-926c-afbed2605bd9.png">